x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/lens_attributes.ts

```diff
@@ -396,10 +399,14 @@ export class LensAttributes {
     }
   }
 
-  getMainYAxis(layerConfig: LayerConfig) {
+  getMainYAxis(layerConfig: LayerConfig, layerId: string, columnFilter: string) {
```

Comment:
I don't think I fully understand the idea of a mainYAxis and a childYAxis of yet. Can you explain to me how they relate to one another?

Reply:
Sorry again for confusing naming here. Seems like i need to clean up this part as well. 

So as far as Lens is concerned there are no main/child axis. I coined these because i thought we will have a main series concept in lens. 

Essentially child columns here means there are more than one column in the layer.